recipe/conda_build_config.yaml

```diff
@@ -1,2 +1,3 @@
-c_stdlib_version:              # [linux]
-  - 2.17                       # [linux]
\ No newline at end of file
+# use a docker image with glibc 2.28 in it for testing
+docker_image:                                  # [x86_64 and linux64]
```

Comment:
As said before, this is should be done via `c_stdlib_version` and not the docker image directly.

Reply:
I reverted this change too. I tried to follow the suggestions from https://conda-forge.org/news/2024/03/24/stdlib-migration/ but no success so far. 

Do you know where would be the right place for me to place the `c_stdlib_version`?

Currently I'm getting the following error in the build.

```
 conda_libmamba_solver.conda_build_exceptions.ExplainedDependencyNeedsBuildingError: Unsatisfiable dependencies for platform linux-64: {MatchSpec("c_linux-64=2.17")}
    Encountered problems while solving:
      - nothing provides requested c_linux-64 2.17.*
    
    Could not solve for environment specs
    The following package could not be installed
    └─ c_linux-64 2.17.*  does not exist (perhaps a typo or a missing channel).
```

Reply:
You need to rerender the recipe for this to take effect.

Reply:
I didn't know I should do that, thanks! 

I updated `c_stdlib_version` from 2.17 to 2.28 and run a rerender, but it still gives me the original error. 

I expected that after updating to 2.28 and running a rerender the file `.ci_support/linux_64_.yaml` would get updated to use the image `quay.io/condaforge/linux-anvil-alma-x86_64:8`. Is that the right assumption?

fyi: I previously set the docker image directly based the implementation done at the [onednn-feedstock](https://github.com/conda-forge/onednn-feedstock/blob/main/recipe/conda_build_config.yaml#L3) repo